releng/sap-cf-base/src/main/java/org/eclipse/dirigible/cf/CloudFoundryModule.java

```diff
@@ -41,12 +43,18 @@ public class CloudFoundryModule extends AbstractDirigibleModule {
 
 	private static final String ERROR_MESSAGE_NO_XSUAA = "No XSUAA service instance is bound";
 	private static final String WARN_MESSAGE_NO_DESTINATION = "No Destination service instance is bound";
+	private static final String WARN_MESSAGE_NO_CONNECTIVITY = "No Connectivity service instance is bound";
 
 	public static final String DIRIGIBLE_DESTINATION_CLIENT_ID = "DIRIGIBLE_DESTINATION_CLIENT_ID";
 	public static final String DIRIGIBLE_DESTINATION_CLIENT_SECRET = "DIRIGIBLE_DESTINATION_CLIENT_SECRET";
 	public static final String DIRIGIBLE_DESTINATION_URL = "DIRIGIBLE_DESTINATION_URL";
 	public static final String DIRIGIBLE_DESTINATION_URI = "DIRIGIBLE_DESTINATION_URI";
 
+	public static final String DIRIGIBLE_CONNECTIVITY_CLIENT_ID = "DIRIGIBLE_CONNECTIVITY_CLIENT_ID";
+	public static final String DIRIGIBLE_CONNECTIVITY_CLIENT_SECRET = "DIRIGIBLE_CONNECTIVITY_CLIENT_SECRET";
+	public static final String DIRIGIBLE_CONNECTIVITY_URL = "DIRIGIBLE_CONNECTIVITY_URL";
+	public static final String DIRIGIBLE_CONNECTIVITY_URI = "DIRIGIBLE_CONNECTIVITY_URI";
```

Comment:
Please add these environment variables to the dirigible's help page for environment variables (https://www.dirigible.io/help/setup/setup-environment-variables/). This could be done after the PR is merged.

Reply:
https://github.com/dirigible-io/dirigible-io.github.io/pull/72